common-arcana.lic

```diff
@@ -122,7 +122,7 @@ module DRCA
     case match
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
-      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm, data['runestone_name'], data['runestone_tm'])
+      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm, runestone_name, runestone_tm = false)
```

Comment:
This is a function call, not definition. ❓ What does `runestone_tm = false` do in this context?

Reply:
I ran a quick test and I'm pretty sure this will force `runestone_tm` to always be false. Here's what I tested with:

```
def foo(a = 1)
	puts "a: #{a}"
end

my_a = 2
foo(my_a = 3)
```

> a: 3

Reply:
> We want to keep it false if the tattoo_tm is triggered. If you look at the previous statement before I changed it, it passes tattoo_tm back through the def to make tattoo_tm true. I do not want to change runestone_tm to true if the match for the tattoo triggers.

This is the "Your desire to prepare this offensive spell suddenly slips away" case. I don't see what it has to do with tattoo matches/triggers. If we need to re-prepare the spell, shouldn't we call the function with the exact same parameters again?